JSDemos/Demos/Charts/SimpleArray/description.md

```diff
@@ -1 +1,3 @@
-In&nbsp;this demo, the Chart component is&nbsp;populated with data taken from a&nbsp;simple JavaScript array. It&nbsp;is&nbsp;the easiest way to&nbsp;provide data for the component, and it&nbsp;is&nbsp;the right choice if&nbsp;you do&nbsp;not need to&nbsp;get data from the server or&nbsp;update data dynamically.
\ No newline at end of file
+You can populate the Chart component with data locally or remotely. If you do not need to get data from the server or update data dynamically, bind the component to a local JavaScript array. To bind Chart to a local array, pass this array to the Chart's [dataSource](/Documentation/ApiReference/UI_Components/dxChart/Configuration/#dataSource) property. 
+
+Then, specify the [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/) type. Each [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/) type has the [argumentField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#argumentField) and the [valueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#valueField) properties. If you specify these properties, the component can determine which object fields in the array indicate Chart arguments and values. The default values for these properties are `arg` and `val` respectively. You can use these data field names to structure your array as shown in this demo.
```

Comment:
I don't like the sentences that simply start with "then". A simple "then" is implied if you describe a sequence of actions. You can be specific, or simply omit "then".

I also tried to simplify / shorten the link between first and second sentences .

```suggestion
Once you assign the data source, specify the [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/) type and its nested options:  [argumentField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#argumentField) and [valueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#valueField). If you specify these properties, the component can determine which object fields in the array indicate Chart arguments and values. The default values for these properties are `arg` and `val` respectively. You can use these data field names to structure your array as shown in this demo.
```